NineAnimator/Models/Anime Source/animeunity.it/AnimeUnity+Episode.swift

```diff
@@ -28,40 +27,13 @@ extension NASourceAnimeUnity {
             episodePageUrl in self
                 .requestManager
                 .request(url: episodePageUrl, handling: .browsing)
-                .responseString
+                .responseVoid
```

Comment:
Is this necessary? Seems like the results from the request are abandoned. You should directly return the `Episode` object if this network request is not necessary.

Reply:
Correct, removed it